README.md

```diff
@@ -18,69 +18,64 @@ The API keys will allow you to authenticate and interact with the APIs of your L
-
-Instead, a summary of the evaluation results can _optionally_ be presented:
-
-- as an artifact within the CircleCI User Interface
-- as a comment on the corresponding GitHub pull request (requires a [GitHub Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens))
-
-### Orb Parameters
-
-The [evals orb](https://circleci.com/developer/orbs/orb/circleci/evals) accepts the following parameters:
+## Getting started
 
-_Some of the parameters are optional based on the eval platform being used._
+Fork this repo to run evaluations on a LLM-based application using the [evals orb](https://circleci.com/developer/orbs/orb/circleci/evals).
 
-#### Common parameters
+This repository includes evaluations that can be run on two evaluation platforms: [Braintrust](https://www.braintrustdata.com/) and [LangSmith](https://smith.langchain.com/). Each example folder contains instructions and sample code to run evaluations.
 
-- **`circle_pipeline_id`**: CircleCI Pipeline ID
+### Here's the process...
 
-- **`cmd`**: Command to run the evaluation
+1. Enter your credentials into CircleCI, which get stored as environment variables on a new context.
+2. Update the CircleCI configuration file with your newly-created context.
+3. Select the evaluation platform where you want to run evaluations.
 
-- **`eval_platform`**: Evaluation platform (e.g. `braintrust`, `langsmith` etc.; default: `braintrust`)
+### 1. Enter credentials into CircleCI
 
-- **`evals_result_location`**: Location to save evaluation results (default: `./results`)
+Entering your OpenAI, Braintrust, and LangSmith credentials into CircleCI is easy.
 
-#### Braintrust-specific parameters
+Just navigate to `Project Settings` > `LLMOps` and fill out the form by Clicking `Set up Integration`.
 
-- **`braintrust_experiment_name`** _(optional)_: Braintrust experiment name
-  - If no value is provided, an experiment name will be auto-generated based on an MD5 hash of `<CIRCLE_PIPELINE_ID>_<CIRCLE_WORKFLOW_ID>`.
+![Create Context](images/create-context.png)
 
-#### LangSmith-specific parameters
+This will create a context with environment variables for the credentials you've set up above.
 
-- **`langsmith_endpoint`** _(optional)_: LangSmith API endpoint (default: `https://api.smith.langchain.com`)
+_:bulb: Please take note of the generated context name (e.g. `ai-llm-eval-examples`). This will be used in the next step to update `context` value in the CircleCI configuration file._
 
-- **`langsmith_experiment_name`** _(optional)_: LangSmith experiment name
-  - If no value is provided, an experiment name will be auto-generated based on an MD5 hash of `<CIRCLE_PIPELINE_ID>_<CIRCLE_WORKFLOW_ID>`.
+![LLMOps Integration Context](images/LLMOps-Integration-Context.png)
```

Comment:
After this line, i think we should also add about providing `GITHUB_TOKEN` so user can get the PR comments.